build

```
@@ -160,6 +163,9 @@ def package_docker(config, target, src_dir, iteration, clean=False):
     elif output in ('rpm', 'rpm:bzip2'):
         frpm = '-t rpm --rpm-compression %s' % ('xz' if output == 'rpm' else 'bzip2')
         shell('docker run --rm -v%s:/tgt -w/tgt %s %s' % (os.path.abspath('targets'), fpm_args, frpm))
+    elif output in ('pacman'):
+        fpacman = '-t pacman'
```

```suggestion
        fpacman = '-t pacman --prefix /usr'
```
I'm not sure if this will work, but I'd prefer that rather than duplicating the whole configuration. Alternatively, just use `fpm_args..replace('/usr/local', '/usr')` below -- this is acceptable as this seems to be [required for ArchLinux](https://wiki.archlinux.org/index.php/Arch_package_guidelines#Package_etiquette).

---

many thx for your review. I tested and confirmed that it can working fine